war/pom.xml

```diff
@@ -99,7 +99,7 @@ THE SOFTWARE.
       -->
       <groupId>org.jenkins-ci</groupId>
       <artifactId>winstone</artifactId>
-      <version>5.6</version>
+      <version>5.7</version>
       <scope>test</scope>
```

Comment:
how does this work? winstone is a test dep here?

but this is for the production code isn't it?

Reply:
It is for production. Basically, Winstone classes are just unpacked into the Jenkins WAR classpath (see below in the code). Historical reasons 🤷‍♂